docs/src/constants/redirect-routes.js

```diff
@@ -81,10 +81,13 @@ export default {
   '/components/native/text': ROUTES.TEXT,
   '/components/native/text-input': ROUTES.TEXT_INPUT,
   '/components/native/text-view': ROUTES.TEXT_VIEW,
-  '/components/native/tappable-link-label': ROUTES.TAPPABLE_LINK_LABEL,
+  '/components/native/tappable-link-label': ROUTES.LINK,
```

Comment:
This was correct before

Reply:
This page has been merged with LINK page so needs to redirect as it does not go to the link page only the tappable link page

Reply:
https://backpack.github.io/components/tappable-link-label?platform=ios - current separate page

Reply:
https://backpack.github.io/components/link?platform=ios - will be the new location